src/config/config.rs

```diff
@@ -41,25 +42,45 @@ pub fn config_validator<'a>() -> V::Structure<'a> {
         command_validator()))
 }
 
-fn find_config_path(work_dir: &PathBuf) -> Option<(PathBuf, PathBuf)> {
+fn find_config_path(work_dir: &PathBuf, show_warnings: bool)
+    -> Option<(PathBuf, PathBuf)>
+{
+    fn maybe_print_warning(dir: &Path) {
+        if dir.join("vagga.yml").exists() {
+            writeln!(&mut stderr(), "There is vagga.yml file in the {:?}, \
```

Comment:
We usually to warnings with a `warn!` macro. The positive outcome of using `warn!` macro is that user can silence all warnings or some of them using `RUST_LOG`